File: src/main/kotlin/codeacademy/bookingforum/app/comment/CommentDto.java
package codeacademy.bookingforum.app.comment;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class CommentDto {
    Long id;
    LocalDateTime dateCommented;
    String content;
    Integer likes;
    Integer dislikes;
    Long userId;
    Long postId;
}

File: src/main/kotlin/codeacademy/bookingforum/app/comment/CommentDto.kt
package codeacademy.bookingforum.app.comment

import java.time.LocalDateTime

data class CommentDto(
    val id : Long? = null,
    val dateCommented : LocalDateTime? = null,
    val content : String? = "",
    val postId : Long? = null,
    val userId : Long? = null,
    val likes : Int? = 0,
    val dislikes : Int? = 0,
)